Formula/streamsets-datacollector.rb

```diff
@@ -0,0 +1,60 @@
+class StreamsetsDatacollector < Formula
+  desc "DataOps Platform for Modern Data Movement"
+  homepage "https://streamsets.com"
+  url "https://archives.streamsets.com/datacollector/3.3.0/tarball/streamsets-datacollector-core-3.3.0.tgz"
+  sha256 "c980e4ecca091c8be2d7769d5c0a4a469e8a2096d1f70b1d502536520d6f980f"
+
+  bottle :unneeded
+  
```

Comment:
Some trailing whitespace here now, which needs snipping.

Reply:
Sorry, I'll try to get it right this time

Reply:
No worries! It's usually a case of text editors _trying_ to be helpful but not quite managing it. Thanks for adding the line 😺